Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestManager.java

```diff
@@ -2115,7 +2116,8 @@ private Case openCase() throws CoordinationServiceException, CaseManagementExcep
                             Case.openAsCurrentCase(metadataFilePath.toString());
                         } else {
                             caseDirectoryPath = PathUtils.createCaseFolderPath(rootOutputDirectory, caseName);
-                            Case.createAsCurrentCase(caseDirectoryPath.toString(), caseName, "", "", CaseType.MULTI_USER_CASE);
+                            CaseDetails caseDetails = new CaseDetails(caseName, "", "", "", "", "");
```

Comment:
There is now a CaseDetails constructor that takes only a case name, intended for use here. Please merge in develop again and update this to use that constructor , thanks!